app/test-coordinates/page.tsx

```
@@ -0,0 +1,358 @@
+
+export default function CoordinatesTestPage() {
+  // 이미지 상태
+  const [imageUrl, setImageUrl] = useState<string>('')
+  const [imageDimensions, setImageDimensions] = useState<ImageDimensions | null>(null)
+  const [imageElement, setImageElement] = useState<HTMLImageElement | null>(null)
+  
+  // 선택된 영역들
+  const [selectedAreas, setSelectedAreas] = useState<Array<{
+    id: string
+    name: string
+    absolute: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)